### PR TITLE
chore(tailwind.config.js): set up tailwind config workflow

### PR DESCRIPTION
**Pull request by alabobriggs**
#### What does this PR do
Set up the workflow of using Tailwind with our components and overriding Tailwind default properties.
Also create a demo page that shows how to use the custom classes.

#### Description of Task to be completed
Override the default styling properties provided by tailwind css, add custom styles based on the 
[styleguid](https://www.figma.com/file/wtojDaoY2PJED946ZMIvsX/OSCA-style-guide%2Fpattern) provided. create a demo page to show how generated custom classes can be used

#### How should this be manually tested

1. checkout this branch
2. run `npm run i` & `npm run develop`
3 visit `http://localhost:8000/tailwind-demo-page` when development server has started
4. open `tailwind.config.js` file to see the changes made and open `tailwind-config-demo.js` under pages folder to see how custom classes are used

 #### What are the relevant issues this PR belongs to
#2

 #### Any background context you want to add
no
 #### Screenshots
Screenshot from 2019-12-01 08-50-24

#2

### CODE REVIEW

File: src/pages/tailwind-config-demo.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+              >
+                H5 heading
+              </h5>
+              <h6
+                className = 'text-h6'
+              >
+                H6 heading
+              </h6>
+              <p className = 'text-base'>
+                This is an example praragraph
```
Comment (BolajiAyodeji) [inline src/pages/tailwind-config-demo.js]: Looks like a typo here in `praragraph`

File: src/pages/tailwind-config-demo.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                <div className = 'bg-dark-green w-full h-full'>
+
+                </div>
+              </div>
+            </div>
+          </section>
+         
+          <section>
+
+          </section>
```
Comment (BolajiAyodeji) [inline src/pages/tailwind-config-demo.js]: This `<section>` is empty, any reason?

File: src/pages/tailwind-config-demo.js (hunk 1 above)
Comment (alabobriggs) [reply src/pages/tailwind-config-demo.js]: Fixed the typo

File: src/pages/tailwind-config-demo.js (hunk 2 above)
Comment (alabobriggs) [reply src/pages/tailwind-config-demo.js]: Not really. Wanted to create another component based on the styleguide but the once created already solved the issue. I will remove it

**Comment by BolajiAyodeji** [comment]
Also, this task is more of `chore` rather than a new feature incase of future PR's :).
Thank you and great work! :1st_place_medal:

**Comment by alabobriggs** [comment]
> The configuration looks great, kindly format the demo page to clean whitespaces and we're good to go.

is this the padding around the components. That was is a padding specified on the `layout.js` file. Should I edit that? @BolajiAyodeji

**Comment by alabobriggs** [comment]
> Also, this task is more of `chore` rather than a new feature incase of future PR's :).
> Thank you and great work!

Thank you. I will take note of this in my next PR

**Comment by BolajiAyodeji** [comment]
> > The configuration looks great, kindly format the demo page to clean whitespaces and we're good to go.
> 
> is this the padding around the components. That was is a padding specified on the `layout.js` file. Should I edit that? @BolajiAyodeji

No, I meant the codebase. `Ctrl+Shift+I` should do the trick

**Comment by alabobriggs** [comment]
> > > The configuration looks great, kindly format the demo page to clean whitespaces and we're good to go.
> > 
> > 
> > is this the padding around the components. That was is a padding specified on the `layout.js` file. Should I edit that? @BolajiAyodeji
> 
> No, I meant the codebase. `Ctrl+Shift+I` should do the trick

Oh okay. I will do that now

**Comment by BolajiAyodeji** [comment]
Thanks for this PR @alabobriggs. :+1:
Kindly pick a new ticket and get started, please as we have just limited time :).

**Comment by BolajiAyodeji** [comment]
@allcontributors please add @alabobriggs for code

**Pull request merged**